src/js/components/Tag/Tag.js

```diff
@@ -14,6 +12,7 @@ import { StyledRemoveButton, StyledTagButton } from './StyledTag';
 const Tag = forwardRef(
   ({ name, value, size, onRemove, onClick, ...rest }, ref) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
+    const RemoveIcon = theme.tag.icons.remove;
```

Comment:
Really this should be more tolerant and have a fallback. How about:
```jsx
const RemoveIcon = theme.tag?.icons?.remove || FormClose;
```

Reply:
Added fallback and added a test.